docker/build_fake_dbs.sh

```diff
@@ -1,14 +1,5 @@
 #!/bin/bash
 
-# conda init, activate environment
-# and add current module (icubam) in edit/develop mode
-conda init bash
-conda activate icubam
-
-# launch the server
-pwd
-ls -a
+# launch the DB creation
 echo "Build test db"
-PYTHONPATH=. python scripts/populate_db_fake.py --config=resources/config.toml
-
-cp ./test.db ./db/
+PYTHONPATH=. python scripts/populate_db_fake.py --config="/home/icubam/resources/${ICUBAM_CONFIG_FILE:-icubam.toml}"
```

Comment:
Just curious: why not install the package instead of modifying PYTHONPATH?

Reply:
This assumes you are running from the root of icubam, but IIUC the goal of the script is to run w/out checking out the git repository no?

Reply:
Sorry, this is from within the image, agreed we should ideally just install icubam to the image instead of importing dynamically.

Reply:
Hmm, maybe I missed something but is icubam package available with pip ? (or maybe I'm missing the topics). Anyway, regarding the script above, it should run without the  PYTHONPATH=. already. I'll test and update if that's the case

Reply:
Removed PYTHONPATH (not required inside the docker image)